IDE/Renesas/e2studio/RA6M4/test/src/wolf_client.c

```diff
@@ -192,17 +196,17 @@ void wolfSSL_TLS_client_do(void *pvParam)
        /* Set callback CTX */
         #if !defined(TLS_MULTITHREAD_TEST)
         
-        memset(&guser_PKCbInfo, 0, sizeof(User_SCEPKCbInfo));
+        memset(&guser_PKCbInfo, 0, sizeof(FSPSM_ST));
```

Comment:
Let's also change these `memset` calls to `XMEMSET` for better portability.

Reply:
Thanks, Fixed.